cfgov/unprocessed/apps/research-reports/css/main.less

```diff
@@ -22,12 +22,12 @@
   height: 100%;
   & > ul {
     a {
-      padding-left: 0.75em;
+      padding-left: 1.85em;
```

Comment:
These should probably use the `unit(…)` helper so it's easier to see the desired pixel output at the base font size. E.g. see https://github.com/cfpb/design-system/blob/38096726d51473b98424db3cac76325b0fc462a9/packages/cfpb-core/src/base.less#L346